pkg/cloudprovider/provider/vsphere/helper.go

```diff
@@ -410,6 +411,37 @@ func generateLocalUserdataISO(userdata, name string) (string, error) {
 	return isoFilePath, nil
 }
 
+func detachDisksPVC (virtualMachine *object.VirtualMachine, moVM mo.VirtualMachine) error {
+    // Get VM folder path
+	glog.Infof("VM URL: %v", moVM.Config.Files.VmPathName)
+	vm_dir := filepath.Dir(moVM.Config.Files.VmPathName)
+	glog.Infof("VM folder: %v", vm_dir)
+
+	// Get devices
+    glog.Infof("Getting pvc disks")
```

Comment:
Please use `glog.V(4).Infof` here and for all other log lines you added, because this is debug logging